scipy/interpolate/_aaa.py

```diff
@@ -309,6 +319,71 @@ def _compute_weights(self, f, z, rtol, max_terms):
         self.support_values = fj[i_non_zero]
         self.weights = wj[i_non_zero]
         self.errors = errors[: m + 1]
+        self._points = z
+        self._values = f
+
+    def clean_up(self, cleanup_tol=1e-13):
+        """Automatic removal of Froissart doublets.
```

Comment:
```suggestion
        """Automatic removal of Froissart doublets (spurious pole-zero pairs).
```

Reply:
IMO the original was better (not a blocker).

One other nit: why is a method for manual removal of doubles referred to as the automatic removal?

Reply:
Automatic in the sense that support points are removed automatically to remove Froissat doublets